Gems/ROS2/Code/Source/Lidar/LidarTemplateUtils.h

```diff
@@ -43,6 +43,7 @@ namespace ROS2
         //! @param rotations Rotations as Euler angles in radians to compute directions from.
         //! @param rootRotation Root rotation as Euler angles in radians.
```

Comment:
You changed the parameter, but you missed the change in the documentation, please fix it.

Reply:
Done.